modules/S99_grepit.sh

```diff
@@ -50,7 +50,7 @@ S99_grepit() {
   local GREPIT_RESULTS=0
 
   local MAX_THREADS_S99
-  MAX_THREADS_S99=$((2*"$(grep -c ^processor /proc/cpuinfo || true )"))
+  MAX_THREADS_S99=$((1*"$(grep -c ^processor /proc/cpuinfo || true )"))
```

Comment:
is this consistent?
other modules use local MAX_MOD_THREADS

Reply:
good point ... will change this